platform/node/src/node_map.cpp

```diff
@@ -25,6 +25,15 @@ struct NodeMap::RenderOptions {
     unsigned int width = 512;
     unsigned int height = 512;
     std::vector<std::string> classes;
+    bool debugTileBorders = false;
```

Comment:
To `mbgl` stuff out of `NodeMap::ParseOptions`. Should I switch it?

I replaced these three bools with the nested debug struct and forgot to remove them.